Yari NPM publish broken on master

From what I can see it looks like the underlying error is:

> ReactDOMServer does not yet support Suspense.
https://reactjs.org/docs/error-decoder.html/?invariant=294

Here is the partial log from GitHub Actions:

```
Building SPAs...
Wrote /home/runner/work/yari/yari/client/build/en-us/_spas/404.html
Wrote /home/runner/work/yari/yari/client/build/en-us/search/index.html
Error: Minified React error #294; visit reactjs.org/docs/error-decoder.html?invariant=294 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at e.t.read (/home/runner/work/yari/yari/ssr/dist/main.js:30:14371)
    at t.renderToString (/home/runner/work/yari/yari/ssr/dist/main.js:30:19841)
    at Fe.o.a.createElement.location (/home/runner/work/yari/yari/ssr/dist/main.js:38:36715)
    at Fe (/home/runner/work/yari/yari/ssr/dist/main.js:38:38488)
    at buildOtherSPAs (/home/runner/work/yari/yari/build/cli.js:357:20)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Se._action (/home/runner/work/yari/yari/build/cli.js:407:9)
    at async Se.run (/home/runner/work/yari/yari/node_modules/@caporal/core/dist/index.js:1:27579)

error: Minified React error #294; visit reactjs.org/docs/error-decoder.html?invariant=294 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at async Te._run (/home/runner/work/yari/yari/node_modules/@caporal/core/dist/index.js:1:32257)

error Command failed with exit code 1.
info Visit yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/client/src/app.tsx
+++ b/client/src/app.tsx
@@ -93,7 +93,11 @@ function LoadingFallback({ message }: { message?: string }) {
 export function App(appProps) {
   useDebugGA();
 
-  const homePage = CRUD_MODE ? <WritersHomepage /> : <Homepage {...appProps} />;
+  // When preparing a build for use in the NPM package, CRUD_MODE is always true.
+  // But if the App is loaded from the code that builds the SPAs, then `isServer`
+  // is true. So you have to have `isServer && CRUD_MODE` at the same time.
+  const homePage =
+    !isServer && CRUD_MODE ? <WritersHomepage /> : <Homepage {...appProps} />;
 
   const routes = (
     <Routes>

